Enhance readme (#23)
XCSoar/xcsoar-data-content: README.md

> # xcsoar-data-content [![Travis Build Status](https://img.shields.io/travis/XCSoar/xcsoar-data-content/master.svg)](https://travis-ci.org/XCSoar/xcsoar-data-content)

Reviewer: ```suggestion
# xcsoar-data-content

[![Travis Build Status](https://img.shields.io/travis/XCSoar/xcsoar-data-content/master.svg)](https://travis-ci.org/XCSoar/xcsoar-data-content)
```

> The data here is made available at [download.xcsoar.org](http://download.xcsoar.org).

Reviewer: ```suggestion
The data here is made available at [download.xcsoar.org](http://download.xcsoar.org).

```

It currently maintains:
 * Waypoint files that contain:
   - Turn-points
   - Airfields
   - Out-landing Sites
 * Airspace files:
   - Airspaces that have no other source

## Contributions
Contributions here have to pass:
- A review by another contributer

> - A parsing check by  [aerofiles](https://github.com/Turbo87/aerofiles)

Reviewer 2: ```suggestion
- A parsing check by [aerofiles](https://github.com/Turbo87/aerofiles)
```

They are then uploaded to download.xcsoar.org.

> Please feel free to correct any errors and submit a [pull request](https://help.github.com/en/articles/creating-a-pull-request)!

Reviewer 2: ```suggestion
Please feel free to add missing data and correct errors by submitting a [pull request](https://help.github.com/en/articles/creating-a-pull-request)!
```

# xcsoar-data-content

[![Travis Build Status](https://img.shields.io/travis/XCSoar/xcsoar-data-content/master.svg)](https://travis-ci.org/XCSoar/xcsoar-data-content)
The data here is made available at [download.xcsoar.org](http://download.xcsoar.org).

It currently maintains:
 * Waypoint files that contain:
   - Turn-points
   - Airfields
   - Out-landing Sites
 * Airspace files:
   - Airspaces that have no other source

## Contributions
Contributions here have to pass:
- A review by another contributer
- A parsing check by [aerofiles](https://github.com/Turbo87/aerofiles)

They are then uploaded to download.xcsoar.org.

Please feel free to add missing data and correct errors by submitting a [pull request](https://help.github.com/en/articles/creating-a-pull-request)! When doing so, please write in the comments what the source of the new data is, so it's easy to verify.

## Relation to [xcsoar-data-repository](https://github.com/XCSoar/xcsoar-data-repository)

> There is a sister repository: [xcsoar-data-repository](https://github.com/XCSoar/xcsoar-data-repository) that generates a [repository](http://download.xcsoar.org/repository) file of all the resources available for [xcsoar's](https://xcsoar.org) File Manager.  Files here are included, and any changes to file names / paths need to be also changed / added in that repository.

Reviewer 2: ```suggestion
There is a sister repository: [xcsoar-data-repository](https://github.com/XCSoar/xcsoar-data-repository) that generates a [repository](http://download.xcsoar.org/repository) file of all the resources available for [xcsoar's](https://xcsoar.org) File Manager. Files here are included, and any changes to file names / paths need to be also changed / added in that repository.
```
